### ISSUE TITLE
Add tracing.

### ISSUE THREAD

**Issue by osolomin90** [hdorgeval/playwright-fluent#17]
**Is your feature request related to a problem? Please describe.**
Currently to turn on tracing feature I need to do this via pure playwright. My code:
const story = playwrightFluent
        .withBrowser(props.browser)
        .withOptions({headless: props.isHeadless, slowMo: 50, args: getBrowserArgs(props.browser)})
        .withDefaultWaitOptions({stabilityInMilliseconds: 50, timeoutInMilliseconds: PLAYWRIGHT_TIMEOUT})
        .withDefaultAssertOptions({stabilityInMilliseconds: 100, timeoutInMilliseconds: 3000})
        .recordNetworkActivity({path: props.harPath})
        .WithDialogs();
    await getPage(story).context().tracing.start({screenshots: true, snapshots: true});

**Describe the solution you'd like**
Please add .withTrace() and stopTracing() methods to api and let them use original parameters.

**Comment by hdorgeval** [comment]
Hi @osolomin90, thank you for your feedback ! 
I suggest to change a little bit the wording to be `.withTracing(options)` instead of `.withTrace(options)`.

I do my best to implement this feature asap.

**Comment by hdorgeval** [comment]
Hi @osolomin90, here is a preview of the tracing API:

```js
const p = new PlaywrightFluent();

await p
  .withBrowser(browser)
  .withOptions({ headless: true })
  .withTracing()
  .startTracing({ title: 'my first trace' })
  .navigateTo(url)
  // do some stuff on the opened page
  .stopTracingAndSaveTrace({ path: path.join(__dirname, 'trace1.zip') })
  // do other stuff
  .startTracing({ title: 'my second trace' })
  // do other stuff
  .stopTracingAndSaveTrace({ path: path.join(__dirname, 'trace2.zip') });

**Comment by hdorgeval** [comment]
Hi @osolomin90 , tracing API is now available on the published package v1.45.0.
Feel free to give any feedback if this does not fit your needs.
